requirements/base.in

```diff
@@ -23,6 +23,7 @@ django-better-admin-arrayfield
 
 # API libraries
 drf-extra-fields
+drf-polymorphic
```

Comment:
Currently unused, might not be needed after all

Reply:
I haven't used it so I removed it